site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,91 @@
+
+### Requirements for adding community plugin to the Marketplace
+Every community plugin must fulfill the following checklist to be added to the Marketplace:
+
+**Product Requirements (Checked by a Product Manager)**
+
+1. The plugin is published under an Apache v2 compatible license (e.g. no GPL, APGL). A list of compatible licenses can be found [here](https://apache.org/legal/resolved.html#category-a).
+2. The source code is available in a public git repository.
+3. There is a public issue or bug tracker for the plugin, which is linked in the plugin documentation and linked via `support_url` in the manifest.
+4. The plugin provides detailed usage documentation with at least one screenshot of the plugin in action, list of features and a development guide. This is typically a README file or a landing page on the web. The link to the documentation is set as `homepage_url` in the manifest. A great example is the [README of the GitHub plugin](https://github.com/mattermost/mattermost-plugin-github/blob/master/README.md).
+5. For every release a changelog has to be published, with a link recorded in the `release_notes_url` property of the `plugin.json` manifest.
+6. The plugin has to be out of Beta and be released with at least v1.0.0.
+7. All configuration is accessible via the UI of Mattermost.
+8. The plugin id defined in the manifest must not collide with the id of an existing plugin in the marketplace. It should follow [the naming convention](https://developers.mattermost.com/extend/plugins/manifest-reference/#id).
+
+**Technical Requirements (Checked by developers of the Toolkit or the Integrations team)**
+
+1. The plugin works for 60k concurrent connections and in a high availability environment. (There are currently no tools available to verify this property. Hence, it is checked via code review by a developer)
+2. The plugin logs important events on appropriate log levels to allow system admins to troubleshoot issues.
+
+**Security Requirements (Checked by a member of the security team)**
+
+1. The plugin does not expose a vulnerability.
+2. The plugins provided an email address or a username on the [Community Server](https://community.mattermost) used to report vulnerabilities in the future.
+
+**Functional Requirements (Checked by a QA tester)**
+
+1. The plugin must set a `min_server_version` in the manifest.
+2. The plugin must work on all Mattermost versions greater or equal then `min_server_version`.
+
+Please note, that Mattermost reserves the right to reject any plugin submission from the marketplace.
+
+Given that plugins are currently in Beta and rapid development is still happening, breaking changes especially for the webapp can happen. It is expected from plugin authors to keep up with these changes and fix bugs that may occur. Breaking changes will be mentioned in the "Important Upgrades Notes" section of the [Mattermost Changelog](https://docs.mattermost.com/administration/changelog.html).
```

Comment:
To avoid confusing Mattermost plugin functionality with the plugins themselves, I would recommend rewording the beginning of this to:
> Given that plugin support is currently in Beta[...]